content/iOS/RSFormView/RSFormViewExample.md

```diff
@@ -0,0 +1,310 @@
+# Build an app with RSFormView in Swift
+*[Germán Stábile](https://www.rootstrap.com/tech-blog/) is an iOS developer on the Rootstrap team and recently has started to write about some of the challenges faced during iOS development. You can also follow Germán on [LinkedIn](https://www.linkedin.com/in/german-stabile-61a1b755/).*
+
+Note: this post is meant to help you integrate RSFormView in your app, if you are interested in the motivation and implementation of RSFormView please take a look at the previous article [Link to previous article].
+
+## Creating the project   
+
+ During this article we will be implementing step by step an single view iOS application with RSFormView integrated.
+ 
+ Open Xcode, click on create new Xcode project and select Single View application. 
+ Select a name and a destination for the project, in our case it'll be RSFormViewExample.
+
+In order to integrate RSFormView we need Cocoapods, so we should install cocoapods and add a podfile in our project.
+You can find instructions in cocoapods [guides](https://guides.cocoapods.org/using/using-cocoapods.html).
+Add this line in your podfile `pod RSFormView`, this will bring the last RSFormView available version whenever you run `pod install/update`.
+Now that you have RSFormView installed you should see a workspace in your project folder, open it. 
+
+## Creating your first view controller
+
+Xcode will create a `Main.storyboard` by default, and a `ViewController.swift` class, since we are gonna be doing the integration programatically
+(you can find instructions to integrate RSFormView in a storyboard based application in the [README](https://github.com/rootstrap/RSFormView))
+lets remove `Main.storyboard` and under Deployment Info in your target General tab, add "LaunchScreen" as Main Interface.
+
+![Alt](images/targetSettings.png)
+
+Now lets rename `ViewController.swift` to something more descriptive, perhaps: `FormViewExampleViewController.swift` 
+and write `application(didFinishLaunchingWithOptions:)` in our `AppDelegate.swift`:
+
+``` swift
+  let navigationController = UINavigationController(rootViewController: FormViewExampleViewController())
+  navigationController.setNavigationBarHidden(true, animated: false)
+  window?.rootViewController = navigationController
+  return true
+```
+Here we are setting a `UINavigationController` with our initial view controller as root, hiding the navigation var and setting it to be our
+`window`'s `rootViewController`.
+
+If you run your app now it will display a white screen, so lets add some content to the view controller. 
+
+## Layouting the view controller 
+
+First lets import `RSFormView` in order to get access to the module: add `import RSFormView` at the top of the file. 
+
+Then lets add properties for our UI components.
```

Comment:
"to" our UI components

Reply:
hmm I think we are adding properties "to" our view controller "for" our UI Components